Dockerfile

```diff
@@ -11,3 +13,4 @@ COPY --from=0 /go/src/github.com/openshift/cluster-kube-apiserver-operator/clust
 
 COPY manifests /manifests
 LABEL io.openshift.release.operator true
+ENTRYPOINT ["/usr/bin/cluster-kube-apiserver-operator"]
```

Comment:
@smarterclayton will this break the CVO or CI in any way?

I think it is much nicer for anyone to just run the image and get the operator binary executed automatically than guess the binary (`cluster-kube-apiserver-operator`) name. WDYT?

Reply:
Who runs the operator manually?